app/assets/stylesheets/_screen.scss

```diff
@@ -657,3 +650,43 @@ div#keycut-help {
 .btn-clipboard {
   padding: 4px 12px 5px;
 }
+
+.activity {
+  @extend .story;
```

Comment:
gzip likes repetition in the code because it can remove it using pointers, thus making it smaller. When you use `@extend` it creates giant selectors that can't be made smaller by gzip because there's no repetition.